src/test/java/uk/gov/hmcts/sptribs/recordlisting/LocationServiceTest.java

```diff
@@ -0,0 +1,136 @@
+package uk.gov.hmcts.sptribs.recordlisting;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.ResponseEntity;
+import uk.gov.hmcts.ccd.sdk.type.DynamicList;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.sptribs.recordlisting.model.HearingVenue;
+import uk.gov.hmcts.sptribs.recordlisting.model.Region;
+
+import javax.servlet.http.HttpServletRequest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static uk.gov.hmcts.sptribs.testutil.TestConstants.TEST_AUTHORIZATION_TOKEN;
+import static uk.gov.hmcts.sptribs.testutil.TestConstants.TEST_SERVICE_AUTH_TOKEN;
+
+@ExtendWith(MockitoExtension.class)
+class LocationServiceTest {
+
+    @InjectMocks
+    private LocationService locationService;
+
+    @Mock
+    private AuthTokenGenerator authTokenGenerator;
+
+    @Mock
+    private ResponseEntity<Region[]> regionResponseEntity;
+
+    @Mock
+    private ResponseEntity<HearingVenue[]> hearingVenueResponseEntity;
+
+    @Mock
+    private HttpServletRequest httpServletRequest;
+
+    @Mock
+    private LocationClient locationClient;
+
+    @Test
+    void shouldPopulateRegionDynamicList() {
+        //When
+        when(authTokenGenerator.generate()).thenReturn(TEST_SERVICE_AUTH_TOKEN);
```

Comment:
Can you make the comments //Given //When //Then in this class, as you've done above in CaseworkerRecordListingTest please? Makes it more readable if we use a consistent standard.

Reply:
Done.